packages/babel-helper-create-class-features-plugin/src/fields.js

```diff
@@ -241,18 +246,28 @@ const privateNameHandlerSpec = {
 };
 
 const privateNameHandlerLoose = {
-  handle(member) {
+  get(member) {
```

Comment:
Out of curiosity, is this small refactoring needed to implement this feature?

(I like the new code, I'm not asking to remove it)

Reply:
Yes. The old code didn't actually use `memberExpressionToFunctions` (because it was unnecessary), and the optional behavior is implemented inside `memberExpressionToFunctions`. These changes are what's necessary for that API.